content/kubermatic/main/tutorials-howtos/cluster-backup/_index.en.md

```diff
@@ -0,0 +1,134 @@
++++
+title = "Integrated User Cluster Backup"
+date = 2023-02-20T12:00:00+01:00
+weight = 3
+enterprise = true
++++
+
+## [Experimental] Integrated User Cluster Backup
```

Comment:
This headline isn't needed (see the [preview](https://deploy-preview-1636--cranky-newton-4e6ed2.netlify.app/kubermatic/main/tutorials-howtos/cluster-backup/)) since the `title` is already used as a page headline.

To mark this as experimental, you could instead do something similar to what was done on the "Encryption at Rest" page: https://github.com/kubermatic/docs/blob/master/content/kubermatic/main/tutorials-howtos/encryption-at-rest/_index.en.md?plain=1#L8

Removing this headline also means updating the following headlines to one level higher ( `##` instead of `###`).